gsa/src/gmp/models/cpe.js

```diff
@@ -22,24 +22,36 @@ import {map} from 'gmp/utils/array';
 
 import Info from './info';
 
-import {parseSeverity, parseDate} from 'gmp/parser';
+import {parseScoreToSeverity, parseSeverity, parseDate} from 'gmp/parser';
 
 class Cpe extends Info {
   static entityType = 'cpe';
 
+  static parseObject(object) {
+    const ret = super.parseObject(object);
+
+    ret.severity = parseScoreToSeverity(ret.score);
+    delete ret.score;
```

Comment:
I am really not sure about this at all. The severity will be only the displayed value of the score in future. Maybe we also switch to display the score directly. So deleting the score feels wrong to me but for the moment it seems the best to do.

Reply:
Yeah I introduced the function that itself cares about dividing the score.
I am with you that we could just show the integer directly ...